### ISSUE TITLE
Wrong version printed

### ISSUE THREAD

**Issue by enna** [leboncoin/morphlingjs#4]
Hello,
When I do `morphling --version `, I always get `0.0.1` also I installed morphlingjs@0.1.99
;)

**Comment by vroudge** [comment]
Yeah I'm aware of that. You can submit a PR if you like to fix it :)


Valentin Roudge
Co-founder / CTO
+33 6 51 48 31 28
www.180gram.io


2017-11-07 11:54 GMT+01:00 Anne <notifications@github.com>:
